scripts/load_tester/drill_config.yml

```diff
@@ -0,0 +1,24 @@
+concurrency: 4
+base: 'http://localhost:8000'
+iterations: 5
+
+plan:
+  - name: Fetch balances
+    request:
+      url: /v1/safes/{{ item.safe_address }}/balances/USD
```

Comment:
I would load the transaction lists too, as these are the ones that really put entries into the cache